AdminDashboard/Controllers/AccountController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using AdminDashboard.BusinessLogicOrchestrators.AccountOrchestrator;
+using AdminDashboard.Models.JsonRequests;
+using AdminDashboard.Repositories.AccountRepository;
+using AdminDashboard.Validators.Accounts;
+using AdminDashboard.Validators.Plans;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace AdminDashboard.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class AccountController : ControllerBase
```

Comment:
The web API is concerned with validation and passes business logic computations to orchestrators!

Reply:
I could find any business logic in `AccountOrchestrator`. It has a few wrappers on top of `AccountRepository` and Web API has references to both of them.

Reply:
Yeah good point. I just injected the repository for some endpoints because there were currently no business logic. I'll do that for all the others, and get rid of the orchestrator. I just added an orchestrator layer in case I wanted to add more functionality in the future, but it's a good point that in the scope of this project, the orchestrators is overkill!